### PR TITLE
Add logout to CustomerLogin service

### PR DESCRIPTION
**Pull request by kwreczycki**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | yes/no
| Fixed tickets | #...   

Hi Dominik,
I think that alternatively to loginCustomer in CustomerLoginService we should have also possibility to logout him.
What do you think about that ?

### CODE REVIEW

File: src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginServiceInterface.php [hunk 1]
```diff
@@ -20,4 +20,6 @@
      * @param CustomerInterface $customer
      */
     public function loginCustomer(CustomerInterface $customer);
+
+    public function logoutCustomer();
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginServiceInterface.php]: missing phpdoc

File: src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginService.php [hunk 2]
```diff
@@ -36,4 +44,10 @@ public function loginCustomer(CustomerInterface $customer)
         $token = new UsernamePasswordToken($customer, null, 'coreshop_frontend', $customer->getRoles());
         $this->securityTokenStorage->setToken($token);
     }
+
+    public function logoutCustomer()
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginService.php]: missing phpdoc

File: src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginService.php [hunk 3]
```diff
@@ -36,4 +44,10 @@ public function loginCustomer(CustomerInterface $customer)
...
+    {
+        $this->securityTokenStorage->setToken(null);
+        $this->requestStack->getCurrentRequest()->getSession()->invalidate();
```
Comment (dpfaffenbauer) [inline src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginService.php]: I am not sure about that. Why do you even need that anyway? Couldn't you just call the logout url directly?

Comment (dpfaffenbauer) [reply src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginService.php]: I mean, that is something the symfony firewall handles

**Comment by dpfaffenbauer** [comment]
Also, this PR targets the 2.0 release, can you rebase to 2.0 pls?

File: src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginService.php (hunk 3 above)
Comment (kwreczycki) [reply src/CoreShop/Bundle/CoreBundle/Customer/CustomerLoginService.php]: Yeah, maybe you have right. I wanted add this method only to have consistent behaviour for this service. If we have login we expect logout also but in fact it's bit overengineer.

**Comment by dpfaffenbauer** [comment]
I guess we can close that.

**Pull request closed without merging**